README.md

```diff
@@ -43,7 +43,7 @@ Create a controller that looks like this, customize to your own situation:
 
 ``` ruby
 class SamlIdpController
```

Comment:
Since this example is for a rails controller. Should we make this class inherit from `ApplicationController`.

.i.e.

```ruby
class SamlIdpController < ApplicationController
```